covid_xprize/validation/tests/test_scenario_generator.py

```diff
@@ -105,10 +105,12 @@ def _check_counterfactual(self, scenario, scenario_npis):
         pd.testing.assert_frame_equal(after_day_npis_truth, after_day_npis_df, "Not the expected unfrozen NPIs")
 
     def test_generate_scenario_future_freeze(self):
```

Comment:
The changes are for these `_future_` tests that are using `self._check_future`: their end date is supposed to be in the future but it wasn't anymore. All the following changes in this file are more or less the same.